docs/clusterdefinition.md

```diff
@@ -454,6 +454,9 @@ A cluster can have 0 to 12 agent pool profiles. Agent Pool Profiles are used for
 |adminUsername|yes|describes the username to be used on all linux clusters|
 |ssh.publicKeys.keyData|yes|The public SSH key used for authenticating access to all Linux nodes in the cluster.  Here are instructions for [generating a public/private key pair](ssh.md#ssh-key-generation).|
 |secrets|no|specifies an array of key vaults to pull secrets from and what secrets to pull from each|
+|customsearchdomain.name|no|describes the search domain to be used on all linux clusters|

```

Comment:
Let's use the exact case here, e.g., `customSearchDomain.name`

Thanks for including docs!